apps/desktop-wallet/src/modals/SettingsModal/DevToolsSettingsSection.tsx

```diff
@@ -108,6 +105,12 @@ const DevToolsSettingsSection = () => {
     sendAnalytics({ event: 'Requested testnet tokens' })
   }
 
+  const openCallContractModal = () =>
+    dispatch(openModal({ name: 'SendModalCallContract', props: { initialTxData: { fromAddress: defaultAddress } } }))
```

Comment:
Should we keep "Modal" as a suffix to help with understanding the file structure better in a glance? Like `SendCallContractModal` 🤔

Reply:
You're absolutely right

Reply:
Done!